src/ports/server/component.ts

```diff
@@ -38,15 +40,49 @@ export async function createServerComponent({
       delete sockets[socket.id]
     })
 
-    socket.on('message', (message: Message) => {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    socket.on('message', (socketMsg: any) => {
```

Comment:
Because the message could be anything, receiving it already casted may cause miss types given that what is received in the message is not an actual Message